README.md

```diff
@@ -213,28 +101,30 @@ be easily converted back to a Pandas DataFrame by using `df_pandas = df_polars.t
 mf = MetaFrame.fit_dataframe(df)
 ```
 
-> Note: if at this point you get the following warning about a potential unique variable, do not worry, it is safe to continue.
+> Note: If you encounter a warning about `PassengerId` not being set as unique, you can safely ignore it and proceed. This warning occurs because `PassengerId` appears to contain unique values, but is not explicitly marked as a unique column. To remove the warning, you can set `PassengerId` to be a unique column. Our documentation explains how to do this when generating Metaframes: [Set Columns as Unique](https://metasynth.readthedocs.io/en/latest/usage/generating_metaframes.html#optional-parameters).
```

Comment:
In this case you will always get the warning, right? That should then be reflected in this sentence. I think it can be shorter like in the original too. The link is really nice though!